src/senaite/core/browser/fields/record.py

```diff
@@ -19,7 +19,6 @@
 # Some rights reserved, see README and LICENSE.
 
 import six
-from types import ClassType
```

Comment:
this slipped in here: https://github.com/senaite/senaite.core/pull/2087#discussion_r934924827